openpathsampling/engines/trajectory.py

```diff
@@ -290,6 +290,49 @@ def next(self):
 
         return ObjectIterator()
 
+    def index_symmetric(self, value):
```

Comment:
Could you add a brief comment to the docstring as to why one would want this? I'm not sure what this would be needed for.